README.md

```diff
@@ -210,6 +210,39 @@ Command: <code>python3 -m pip install --user --upgrade pip</code>
 	Link to Solution: <a href="https://github.com/Udayraj123/OMRChecker/issues/70#issuecomment-1268094136">#70</a>
 </details>
 
+### Running in a Container Runtime (Docker or Podman)
+
+The repo contains a containerfile that can be used to run ORM checker without 
+interfering with a system python installation.
+The following instructions are based on using `docker` but can be adapted to
+`podman` by exchanging the names of the two programs.
```

Comment:
Apologies if I sound opinionated, but let's keep it simple for general audience and prefer keeping Docker instructions. 

Those who use Podman would already know the translation of these steps. Either remove this line, or if you want, we can put this in the wiki instead and link it in Readme